Gemfile

```diff
@@ -55,4 +55,5 @@ end
 
 group :development do
   gem 'grape_on_rails_routes', '~> 0.3.2'
+  gem 'figaro'
```

Comment:
we need it to setup env variables in development. It would be much easier than writing multiple `export VAR` commands. `application.yml` is in gitignore, because we don't need share it. Devops use their own env solution